### PR TITLE
Add Docs for Establishing an Identity

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Add Docs for Establishing an Identity to Block Producer Onboarding section.

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

**Review by eshaben — COMMENTED** [review]
Awesome! This looks great.

I think this will be kept as a general end-user. page, so can you please go through and remove anything specifically targeting block producers? As for where it will live exactly, that is still TBD. There is a meeting tomorrow, so I think we should be nailing this down soon. Will keep you posted! 🙂 

Since this will be more general purpose, you can also add sections on removing an identity.

The line-width on the icon looks a bit thin compared to the other ones, can you please check it?

**Comment by eshaben** [comment]
Oh, I ran this through grammarly and there are a few suggestions that could be applied and also a really strange thing: sometimes we use `’` (curlies) and other times we use `'` (straight) haha

**Comment by themacexpert** [comment]
> Oh, I ran this through grammarly and there are a few suggestions that could be applied and also a really strange thing: sometimes we use `’` (curlies) and other times we use `'` (straight) haha

fixed to straight for all

**Comment by eshaben** [comment]
> fixed to straight for all

thank you! last thing is this comment:

> Since this will be more general purpose, you can also add sections on removing an identity.

can you please add these sections?

**Comment by themacexpert** [comment]
added Clear identity section

File: builders/account-management/identity.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+To clear your identity, take the following steps from Extrinsics tab of the Polkadot.js Apps UI: 
+
+1. Select your account from the **using the selected account** dropdown
+2. Select **identity** from the **submit the following extrinsic** dropdown
+3. Then select the **clearIdentity()** function
+4. Click **Submit Transaction**
+
+![Clear an identity](/images/builders/account-management/identity/identity-8.webp)
+
+You will then be prompted to sign the transaction. If everything looks good, you can enter your password and click Sign and Submit to sign and send the transaction. You should see status notifications pop-up in the top right hand corner confirming the transaction.
```
Comment (eshaben) [inline builders/account-management/identity.md:152]:
```suggestion
You will then be prompted to sign the transaction. If everything looks good, enter your password and click **Sign and Submit** to sign and send the transaction. You should see status notifications in the top right-hand corner confirming the transaction.
```

File: builders/account-management/identity.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+3. Select your account
+4. Click the **+** button to get your identity information
+
+![Request identity information](/images/builders/account-management/identity/identity-7.webp)
+
+You can see now that you've successfully set an identity! Once you clear your identity, the deposit in your reserved balance will get transferred back to your transferable balance. If you need to change your identity, you can go through the process of setting your identity again. Please note that you must ensure all fields are re-entered, even if only one field needs to be changed, or they will be overwritten. You will not need to pay another deposit unless custom fields are used, but you will need to pay gas fees.
+
+## Clear an Identity {: #confirm-identity-extrinsics }
+
+To clear your identity, take the following steps from Extrinsics tab of the Polkadot.js Apps UI:
```
Comment (eshaben) [inline builders/account-management/identity.md:143]:
```suggestion
To clear your identity, take the following steps from **Extrinsics** tab of the Polkadot.js Apps UI: 
```

File: builders/account-management/identity.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+Finally, once all of your identity information has been added, you can scroll to the bottom of the page and click **Submit Transaction**.
+
+You will then be prompted to sign the transaction. Remember, an additional deposit is required for each additional custom field. If everything looks good, enter your password and click **Sign and Submit** to sign and send the transaction.
+
+You should see status notifications pop up in the top right-hand corner confirming the transaction. If successful, you've set an identity! Congratulations! To ensure everything went through and your identity information looks good, you can verify your identity.
+
+### Confirm an Identity {: #confirm-identity-extrinsics }
+
+To verify the addition of your identity information, you can click on the **Developer** tab and then navigate to [Chain state](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}.
```
Comment (eshaben) [inline builders/account-management/identity.md:128]:
```suggestion
To verify the addition of your identity information, you can click on the **Developer** tab and then navigate to [**Chain state**](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/chainstate){target=\_blank}.
```

File: builders/account-management/identity.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+If the identity information matches what you entered, you've successfully set an identity!
+
+Once you clear your identity, the deposit in your reserved balance will get transferred back to your transferable balance. If you need to change your identity, you can go through the process of setting your identity again. Please note that you must ensure all fields are re-entered, even if only one field needs to be changed, or they will be overwritten. You will not need to pay another deposit unless custom fields are used, but you will need to pay gas fees.
+
+## Manage an Identity via Extrinsics {: #manage-via-extrinsics }
+
+### Set an Identity {: #set-identity-extrinsics }
+
+To register an identity using the extrinsics UI, navigate to the [Extrinsics page](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/extrinsics){target=\_blank} on Polkadot.js Apps. Please ensure your input does not exceed 32 characters for each identity field. To complete your identity, take the following steps:
```
Comment (eshaben) [inline builders/account-management/identity.md:93]:
```suggestion
To register an identity using the extrinsics UI, navigate to the [**Extrinsics** page](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/extrinsics){target=\_blank} on Polkadot.js Apps. Please ensure your input does not exceed 32 characters for each identity field. To complete your identity, take the following steps:
```

File: builders/account-management/identity.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+If you want to add custom fields beyond the default fields, follow the instructions for [Managing an Identity via the Extrinsics UI](#manage-via-extrinsics).
+
+!!! note
+    Please note that using the **Accounts** UI on Polkadot.js Apps is recommended to manage your identity as it provides an easy-to-use interface that enforces character limits. If you use the **Extrinsics** UI, please be aware that your input for each field (i.e., name, email, etc.) must be 32 characters or less; otherwise, your information will be cut off.
+
+## Manage an Identity via Accounts {: #manage-via-accounts }
+
+### Set an Identity {: #set-identity-accounts }
+
+To get started with setting an identity using the Accounts UI, head to the [Accounts tab](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/accounts){target=\_blank} on the Polkadot.js Apps explorer.
```
Comment (eshaben) [inline builders/account-management/identity.md:56]:
```suggestion
To get started with setting an identity using the Accounts UI, head to the [**Accounts** tab](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/accounts){target=\_blank} on the Polkadot.js Apps explorer.
```

**Review by eshaben — COMMENTED** [review]
just noticed a few items that should be in bold

**Review by eshaben — APPROVED** [review]
LGTM! Great job! ✨

**Pull request merged**